Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "dusk-plonk"
-version = "0.10.0"
+name = "plonkup"
```

Comment:
I will change the commit message to:

```
Change crate name from `dusk-plonk`  to `plonkup`
```

If you want to give context, use the commit's message body:

```
Change crate name from `dusk-plonk`  to `plonkup`

This is needed to split the plonk and plonkup code base
```